sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -99,8 +99,8 @@ class Message(object):  # pylint: disable=too-many-public-methods,too-many-insta
 
     """
 
-    def __init__(self, body, **kwargs):
-        # type: (Union[str, bytes], Any) -> None
+    def __init__(self, body=None, **kwargs):
```

Comment:
I would not actually make Body None by default.

The allowance here to my eyes is that you should be ABLE to make it None, not that it should always be none.  I still consider it a useful guardrail that a user should provide _something_, if only to show intent.

Let me know what you think, as always feel encouraged to throw rocks etc.

Reply:
to your point @KieranBrantnerMagee, I guess we even don't need to mention it as a new feature in changelog cuz it would look like "we're promoting body can be None".

Reply:
i can just silently allow passing None - sounds fair to me 
do you think we should instead log a warning if the payload is none?

Reply:
I probably wouldn't, because as the whole thrust of this issue, None is "Valid" as far as the service is concerned.  Apparently some users even rely on it, so warning them on it might be a bit weird.